pkg/controllers/repositoryserver/repositoryserver_controller.go

```diff
@@ -76,6 +77,8 @@ func (r *RepositoryServerReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	logger.WithValues("Name", repositoryServer.Name).WithValues("Namespace",repositoryServer.Namespace)
```

Comment:
@akankshakumari393 did you run the controller and see how the logs are seen to the user? I think the logs that get printed already have name and namespace of the repository server CR. 
eg:
```
{"level":"debug","ts":"2023-07-04T15:45:05Z","msg":"Update pod name in RepositoryServer /status","controller":"repositoryserver","controllerGroup":"cr.kanister.io","controllerKind":"RepositoryServer","RepositoryServer":{"name":"kopia-repo-server","namespace":"kanister"},"namespace":"kanister","name":"kopia-repo-server","reconcileID":"b5e5030d-0665-4c71-a8eb-78de738c8f0d"}
{"level":"debug","ts":"2023-07-04T15:45:05Z","msg":"Fetch latest version of RepositoryServer to update the ServerInfo in its status","controller":"repositoryserver","controllerGroup":"cr.kanister.io","controllerKind":"RepositoryServer","RepositoryServer":{"name":"kopia-repo-server","namespace":"kanister"},"namespace":"kanister","name":"kopia-repo-server","reconcileID":"b5e5030d-0665-4c71-a8eb-78de738c8f0d"}

How does the log look like when you add values explicitly?

Reply:
I checked, they are not adding any value, Earlier I thought those values are because I added it in logs. As we discussed with @PrasadG193, we wanted to add name and namespace of the CR, If that is already there in logs configured, I think this PR would be merged similarly to what we reverted without any change.